app_dart/lib/src/service/scheduler.dart

```diff
@@ -224,11 +224,13 @@ class Scheduler {
       var policy = target.schedulerPolicy;
       // Release branches should run every task
       if (Config.defaultBranch(commit.slug) != commit.branch) {
-        policy = GuaranteedPolicy();
+        policy = const GuaranteedPolicy();
       }
       final priority = await policy.triggerPriority(
-        task: task,
-        datastore: datastore,
+        task.name!,
```

Comment:
I had to check out the PR to see which Task this was and that 'Name'->'name' is the same as `Mac_mokey run_debug_test_android`

Can't wait to be done with the migration.